cumulusci/robotframework/CumulusCI.py

```diff
@@ -245,14 +245,29 @@ def debug(self):
         set_pdb_trace()
 
     def start_perf_timer(self):
-        """Start an elapsed time stopwatch for performance tests"""
+        """Start an elapsed time stopwatch for performance tests.
+
+        Example:
+
```

Comment:
👍  I really appreciate the fact that you're adding examples to the documentation.